tokens/staking/stake.md

```diff
@@ -42,8 +42,10 @@ Token持有者可以向候选人质押自己的Token，这一过程称为委托
 ### 绑定更多Token或减少绑定Token {: #bond-more-or-less }
 
  - **delegatorBondMore**(*address* candidate, *uint256* more) —— 向已经委托的收集人增加质押Token数量的请求。取代已弃用的`nominatorBondMore`函数
- - **scheduleDelegatorBondLess**(*address* candidate, *uint256* less) —— 对已经委托的收集人减少质押Token数量的请求。该数额不得使您的总质押量低于最低委托质押量。在您通过`executeCandidateBondRequest`执行请求前，需经过一个[退出延迟](/learn/features/staking/#quick-reference/#:~:text=减少委托时长){target=_blank}，随后才能执行请求。取代已弃用的`nominatorBondLess`函数
- - **executeCandidateBondRequest**(*address* candidate) —— 对一个特定候选人减少绑定的执行行为。该函数仅用于已计划的绑定请求，且退出已生效之后
+ - **scheduleDelegatorBondLess**(*address* candidate, *uint256* less) —— 对已经委托的收集人减少质押Token数量的请求。该数额不得使您的总质押量低于最低委托质押量。在您通过`executeDelegationRequest`执行请求前，需经过一个[退出延迟](/learn/features/staking/#quick-reference/#:~:text=减少委托时长){target=_blank}，随后才能执行请求。取代已弃用的`nominatorBondLess`函数
+ - **executeDelegationRequest**(*address* candidate) —— 对一个特定候选人减少绑定的执行行为。该函数仅用于已计划的绑定请求，且退出已生效之后
+ - **scheduleCandidateBondLess**(*uint256* less) - 允许收集人候选人请求将其自绑定减少给定数量的extrinsic。在您通过`executeCandidateBondLess`执行请求前，需经过一个[退出延迟](/learn/features/staking/#quick-reference/#:~:text=减少委托时长){target=_blank}，随后才能执行请求。
+ - **executeCandidateBondLess**(*address* 候选人) - 执行减少候选人的自绑定请求。该函数仅用于已计划的绑定减少请求，且退出已生效之后
```

Comment:
this was the closest i could get to commenting on the actual line but for the **scheduleRevokeDelegation** function doesn't look correct, please double check it also **executeDelegationRequest** looks like it was removed from the Revoke Delegations section